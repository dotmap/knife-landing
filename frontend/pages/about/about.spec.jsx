/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import AboutPage from '.'

describe('About page', () => {
  it('matches snapshot', () => {
    const component = renderer.create(<AboutPage />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
