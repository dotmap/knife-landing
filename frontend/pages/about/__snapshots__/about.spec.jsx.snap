// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page matches snapshot 1`] = `
<main>
  <header>
    <h1>
      Next.js Example
    </h1>
  </header>
  <section>
    <p>
      This is another page of the SSR example, you accessed it
       
      <strong>
        client
         side
      </strong>
      .
      <p>
        <p>
          Client: 
        </p>
      </p>
    </p>
    <p>
      You can reload to see how the page change.
    </p>
    <a
      href="/"
      onClick={[Function]}
    >
      Go to Home
    </a>
  </section>
</main>
`;
